pandas/io/excel.py

```diff
@@ -269,6 +290,22 @@ def _get_default_writer(ext):
     return _default_writers[ext]
 
 
+def _is_int_or_intlist(usecols):
+    if isinstance(usecols, int) or (isinstance(usecols, list) and
+                                    all(isinstance(v, int) for v in usecols)):
+        return True
+
+    return False
+
+
+def _is_excel_columns_notation(columns):
+    if isinstance(columns, compat.string_types) and all((x is not "") and
```

Comment:
why are you comparing against an empty string? that is an error. make this as simple as possible. IN fact I would *only* to accept a string here, and not a list of excel notations.